CIP-0116/README.md

```diff
@@ -247,21 +242,57 @@ Instead of providing all three variants of encoding, we base the schema on the o
 
 It is up to implementors to decide how to serialize the values into CBOR. The property we want to maintain is preserved regardless of the choice: for every block binary there is exactly one JSON encoding.
 
+### Versioning
+
+This CIP should not follow a conventional versioning scheme, rather it should be altered via pull request before a hardforks to add new a JSON schema to align with new ledger ers. Each schema must be standalone and not reuse definitions between eras. Authors MUST follow the [Schema Scope](#scope-of-the-schema), [Schema Design Principles](#schema-design-principles) and [Schema Conventions](#schema-conventions).
+
+Furthermore, for each subsequent schema, the [changelog](./changelog.md) must be updated. Authors must clearly articulate the deltas between schemas.
+
 ## Rationale: how does this CIP achieve its goals?
 
-## Path to Active
+### Scope
 
-- [ ] Complete the specification
-- [ ] Provide an implementation of validating functions that uses this json-schema
-- [ ] Collect a list of cardano domain types implementations and negotiate transition to the specified formats with maintainers (if it makes sense and is possible)
+We keep a lean scope, limited to the data types within Cardano blocks. The rationale for this is that block data is by far the most useful for the majority of Cardano actors. There is also one nice benefit that the definitions can map directly from the provided CDDL file from ledger team.
```

Comment:
> We keep a lean scope, limited to the data types within Cardano blocks

It does not sound like a lean scope. It's the most wide scope we could possibly have.

Reply:
oops, I was being silly

Reply:
changed it